Sources/YMDatePicker/YMDatePicker/YMDatePicker.swift

```diff
@@ -123,7 +123,7 @@ extension YMDatePicker: UICollectionViewDataSource , UICollectionViewDelegateFlo
             return  cal.date(byAdding: .day, value: indexPath.section * 7 + row - startDayNumber, to: startDay)!
         } else {
             let year = (indexPath.section / 12) + cal.component(.year, from: Date())
-            let month = indexPath.section + cal.component(.month, from: Date())
+            let month = indexPath.section - (indexPath.section / 12) * 12 + cal.component(.month, from: Date())
```

Comment:
In this situation, you should not change month algorithm. You just have to remove adding year.

Reply:
Oops...